contents/tutorials/css-selectors-for-actions.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Creating actions using CSS selectors
+sidebar: Docs
+showTitle: true
+author: ['hophog']
```

Comment:
@corywatilo Do we have a name for this author? Assuming this is a GH profile it doesn't give me one. 

Need a name so we can give it an authors.json entry, as it doesn't seem to have one.

Reply:
Written by [Igor Kotua](https://www.linkedin.com/in/igorkotua/)